CONTRIBUTING.md

```diff
@@ -290,6 +290,34 @@ When a new release is tagged on the `main` branch (excluding prereleases), it tr
 1. The Docker image is published to GitHub Packages with the version number and `latest` tag.
 2. The deployment is carried out on Langfuse Cloud. This is done by force pushing the `main` branch to the `production` branch during every release, using the [`release.yml`](.github/workflows/release.yml) GitHub Action.
 
+## Theming
+
+At Langfuse, we utilize CSS variables to manage our theme settings across the platform.
+
+Our approach leverages separate CSS variables for backgrounds (--background) and foregrounds (--foreground), fully adhering to the [shadcn/ui](https://ui.shadcn.com/docs/theming) color conventions. We recommend using HSL values for these colors to enhance consistency and customization.
+
+### Adding New Colors
+
+1. Global Definitions: Add new CSS variable definitions in the global.css file.
+2. Tailwind Configuration: Reflect these new colors in the tailwind.config.js to maintain alignment with Tailwind's utility classes.
+
+### Utility Classes
+
+- Border Color: `border`
+- Primary Text: `primary`
+- Secondary Text: `muted-foreground`
+- Hover States: Use `primary-foreground` or `accent`
+- Background: `background`
+- Main Content: `foreground`
+- Primary Color: `primary`
+- Accents: `primary-accent`
+- Selectable Items: `input`
+- Link Background: `muted-indigo`

```

Comment:
can we switch to some template here as well? I though we use some [template]\80 for these

also, other colors (red, yellow, green) missing